client/client-multi/client-objective-c/Kaa/channel/failover/FailoverDecision.h

```diff
@@ -22,6 +22,8 @@
  * Managed by a failover strategy.
  */
 typedef enum {
```

Comment:
While you're modifying this enum anyway, you should change it to Objective-C enum(NS_ENUM)